### DISCUSSION TITLE
Relative path to index.md files not working on pages. Working fine in the navigation

### DISCUSSION THREAD

**Discussion by emign** [mkdocs/mkdocs#2908] [Q&A]
Hi,
I have the following folder structure:

```
Samenpflanzen als Lebewesen/
├─ index.md
├─ Samenpflanzen als.md
Der Mensch als Lebewesen/
├─ index.md
├─ Stoffwechsel/
│  ├─ index.md
├─ Information - Aufnahme, Verarbeitung und Reaktion.md
Ökosystem Grünland.md
```
The contents of `Ökosystem Grünland.md` are:
```
---
---
[[Samenpflanzen als]]

[[Samenpflanzen als Lebewesen]]

[[Der Mensch als Lebewesen]]

[[Stoffwechsel]]

[[Information - Aufnahme, Verarbeitung und Reaktion]]
```

You can open the resulting mkdocs generated html file here: [https://unterricht.emig.me/Unterricht/F%C3%A4cher/Biologie/NuT/%C3%96kosystem%20Gr%C3%BCnland/](https://unterricht.emig.me/Unterricht/F%C3%A4cher/Biologie/NuT/%C3%96kosystem%20Gr%C3%BCnland/)

As you can see, the links to `Samenpflanzen als.md` and `Information - Aufnahme, Verarbeitung und Reaktion.md` are working perfectly fine. But the links to the sections with the `index.md files do not work.

For example the `Stoffwechsel` link is generated as : `<a href="../../../../Stoffwechsel" title="Stoffwechsel">Stoffwechsel</a>`
but it should be: `<a href="../Der%20Mensch%20als%20Lebewesen/Stoffwechsel/">Stoffwechsel</a>`

Please note, that the link in the navigation bar on the left actually IS generated correctly

**Comment by oprypin** [comment]
MkDocs doesn't have the syntax `[[`. You are probably using some plugin but not mentioning it. And the problem seems to be with the plugin.

**Comment by Andre601** [comment]
From what I can see are you using a plugin (EZLinks would be my guess).
In such a case is it 120% caused by the plugin. EZLinks in particular has such issues, as I myself encountered those too.

So better do the links yourself.

Also, just a suggestion, but maybe don't use umlauts in file names.
While browsers can support them, do they often look extremely ugly (As shown by your link), so it's better to use stuff like ue, ae and so on (So `fächer` becomes `faecher`).

**Reply by emign** [reply]
Yes I use Ezlinks but mkdocs creates the navbar correctly from them. Only the page internal links are wrong. So it has to be on the mkdocs side, doesn't it ?
